code/modules/research/techweb/all_nodes.dm

```diff
@@ -1047,6 +1047,7 @@
 	/obj/item/cautery/alien, /obj/item/surgicaldrill/alien, /obj/item/screwdriver/abductor, /obj/item/wrench/abductor, /obj/item/crowbar/abductor, /obj/item/multitool/abductor, /obj/item/stock_parts/cell/infinite/abductor,
 	/obj/item/weldingtool/abductor, /obj/item/wirecutters/abductor, /obj/item/circuitboard/machine/abductor, /obj/item/abductor_baton, /obj/item/abductor, /obj/item/stack/sheet/mineral/abductor)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
+	design_ids = list("icupgclo")
```

Comment:
this is being replaced by design_ids = list("alienalloy") three lines below and this particular design is only INSTANT cloning upgrade, you can still clone without it, it just takes some time

Reply:
Guess I'll need to update PR for an even more comprehensive cloner re-allocation then